src/driver_micarray_alsa.cpp

```diff
@@ -24,13 +24,33 @@
 
 namespace matrix_malos {
 
-bool MicArrayAlsaDriver::ProcessConfig(const DriverConfig& /*config*/) {
-  // TODO (andres.calderon@admobilize.com):  handle SetGain
+bool MicArrayAlsaDriver::ProcessConfig(const DriverConfig& config) {
+  MicArrayParams micarray_config(config.micarray());
+
+  const int16_t gain = static_cast<int16_t>(micarray_config.gain());
+
+  const float azimutal_angle =
+      static_cast<float>(micarray_config.azimutal_angle());
```

Comment:
If these come as float in the protos you don't need the float casts. Otherwise cast is good.